include/godot_cpp/core/class_db.hpp

```diff
@@ -112,6 +112,33 @@ class ClassDB {
 	template <class T, bool is_abstract>
 	static void _register_class(bool p_virtual = false, bool p_exposed = true);
 
+	template <class T>
+	static GDExtensionObjectPtr create(void *data) {
```

Comment:
Even though this is private, I feel like this is too generic of a name, and it could confuse people reading the code later. Maybe:

```suggestion
	static GDExtensionObjectPtr _create_instance_func(void *data) {
```

This name is derived from the property on the struct that we're setting this on.

Godot's `ClassDB` it has a similar-ish function that's called `_create_ptr_func()`, so this would also be a little closer to that style of name.